Agent-skills-antigravity-best-practices.html

```
@@ -0,0 +1,1432 @@
+        <section id="philosophy">
+          <h2><span class="num">02</span>設計思想：Progressive Disclosure（段階的開示）</h2>
+          <p>
+            Agent Skillsの中核にある設計原則が
+            <strong>Progressive Disclosure（段階的開示）</strong>
+            です。Anthropicのエンジニアリングブログでは、これを「よく整理されたマニュアル」に例えています。目次だけをまず読み、必要な章だけを開き、さらに詳細が必要なら巻末の付録を参照する——これと同じ階層構造をコンテキストウィンドウの中で再現します。
+          </p>
+
+          <h3>三段階のロード</h3>
+          <div class="diagram">
+            <div class="mermaid">
+              flowchart LR A["Discovery: 全スキルのdescriptionを事前ロード"] -->
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)